components/ui/VersionFilterControl.vue

```diff
@@ -39,6 +39,24 @@
       placeholder="Filter versions..."
       @input="updateVersionFilters()"
     ></Multiselect>
+    <Multiselect
+      v-if="getValidChannels().length > 1"
+      v-model="selectedChannel"
+      :options="getValidChannels()"
+      :custom-label="
+        (x) => (x === 'any' ? 'Any channel' : $capitalizeString(x))
+      "
+      :multiple="false"
+      :searchable="false"
+      :show-no-results="false"
+      :close-on-select="true"
+      :clear-search-on-select="false"
+      :show-labels="false"
+      :allow-empty="true"
+      class="release-channel"
+      placeholder="Filter channels..."
+      @input="updateVersionFilters()"
+    ></Multiselect>
```

Comment:
How come it's a single select? For consistency + additional functionality I feel like it would be better if you could choose multiple and the ones you want.

Reply:
It just seemed kinda weird as a single select and not really how most would want to use it as it made it harder to switch from one channel to another (say, from showing just Releases to just Alphas), but I can change it if you think I should

Reply:
Changed